### PR TITLE
Kovacs caliber change and improvement to the Kadmin and Vintorez

### PR DESCRIPTION
**Pull request by maikilangiolo**
## About The Pull Request
Changes Kovacs from the .20 marksman role (already done by the vintorez and kadmin) to a .30 battle rifle purpose: hits hard, scope, though high recoil (not as hard as bolt guns since it is magazine fed and semi auto).
Slighly improves the Kadmin and Vintorez as they were seriouly anemic and overall very weak. The stats were low and it was a mistake on my part because I did not take into account the fact both of these guns were _slow firing and thus seriouly under performing_ compared to automatic guns of any other caliber.

the Kovacs was approved by [REDACTED] (who is he and how did he shit my pants????).
image

## Why It's Good For The Game
The kovacs, vintorez and kadmin treated on each other's feet and were altogether too similar (kovacs and vintores in particular). Why have three guns with the same fucking role and purpose? all .20 guns but the wintermute are marskman guns, while the .30 lacks their own scoped battle rifle.

## Testing
It's literally a number change and I have an autistic spreadsheet for number changes.

## Changelog
:cl:
balance: the Kovacs is now .30 caliber and designed to hit hard at the cost of high recoil
balance: improved the Kadmin and Vintorez stats as they were too weak
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm [hunk 1]
```diff
@@ -146,8 +146,8 @@
 	force = WEAPON_FORCE_DANGEROUS // weaker than novakovic, but with a bayonet installed it will be slightly stronger
 	armor_divisor = ARMOR_PEN_GRAZING
 	caliber = CAL_SRIFLE
-	damage_multiplier = 1.7
-	penetration_multiplier = 0.7
+	damage_multiplier = 1.8
```
Comment (DiaFRAME444) [inline code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm:149]: If the Hornet can have Damage Multiplier 2.0 this big two-handed gun can have Damage Mult 2.0.

Comment (maikilangiolo) [reply code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm:149]: naturally I would agree, however whoever before me added all the guns we have now, madde the revolvers vastly better than the bolt guns. the hornet is balance in the revolver category, the kadmin in the bolt gun category. do note that the consul does more damage than the novak/mosin.

I did not change this balance not now and neither when the armor rework was done (as it was not the intention)

File: code/modules/projectiles/guns/projectile/battle_rifle/kovacs.dm [hunk 2]
```diff
@@ -12,20 +12,20 @@
 	origin_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 1)
 	slot_flags = SLOT_BACK
 	load_method = MAGAZINE
-	mag_well = MAG_WELL_RIFLE|MAG_WELL_RIFLE_L
+	mag_well = MAG_WELL_RIFLE
 	magazine_type = /obj/item/ammo_magazine/srifle
```
Comment (Firefox13) [inline code/modules/projectiles/guns/projectile/battle_rifle/kovacs.dm:16]:
```suggestion
	magazine_type = /obj/item/ammo_magazine/lrifle
```

**Review by Firefox13 — CHANGES_REQUESTED** [review]
Don't forget to edit Kovacs disk.

File: code/modules/projectiles/guns/projectile/battle_rifle/kovacs.dm (hunk 2 above)
Comment (maikilangiolo) [reply code/modules/projectiles/guns/projectile/battle_rifle/kovacs.dm]: haha thanks honey

**Comment by Rexdyer1** [comment]
I gotta ask. why are we making the only serb .20 a .30? it is a primarily serb gun. they have a .30 already in their arsenal. is it just because the kovacs spawns more? I say this should be reconsidered. Also just because the kadmin and vintorez exist is not justsification to change it to a .30, because vintorez and kadmin are not serb sspawned guns. you just removed the serbs only .20 rifle from their arsenal.

**Comment by Rexdyer1** [comment]
Also if you want to change a gun from a marksman to a battle rifle might as well go with the vintorez instead of the kovacs, if anything you'd be best to just alter the frozenstar wooden hunting AK into a scoped battle rifle and keeping the kovacs the same so serbians have access to a .20 marksman rifle instead of having them have 2 .30 AK's for their weaponry spawn. its a major balance issue and makes serbian weaponry redundant for one of the servers best antags.

**Comment by Firefox13** [comment]
Kovacs is already a battle rifle

**Comment by Rexdyer1** [comment]
then why is it being made .30?

**Comment by Firefox13** [comment]
Balance I guess

**Comment by maikilangiolo** [comment]
> then why is it being made .30?

There is no in-game difference for the term. In this PR, i chose to divide between marksman rifles and battle rifles. It was made .30 because, as said, the Kovacs, the vintorez and the Kadmin all did the same thing: scoped .20 rifle intended for armor piercing. 
As a general rule, in eris no more than 2 guns do the same thing (i.e. molly and stray light are very low AP but high rof, miller and Havelock are less effective vs armor), though this falls apart in the assault rifle category for ok obvious reasons. 

Anyway, the Kovacs was moved from yet another low recoil, low damage high AP scoped gun (i.e. marksman rifle), to a heavier hitting, high recoil scoped gun (battle rifle), a niche with no other guns (not counting the amr), and also expands the .30 lineup which, out of 9 guns, are all automatic (and in the case of the .20 guns, 3/5 were marksman rifles).

**Comment by Rexdyer1** [comment]
I can understand why you'd want to limit the number of guns that fulfill the same role, however I would like to point out a MAJOR ISSUE. this is a specific antag faction weapon that spawns as part of a niche weapon group that spawns for the Serbians fulfilling a heavy hitting armor piercing role, changing it to a .30 not only removes one of the few .20 of which there are 4 not including the marshall revolver and one-star only gun. You remove a gun that is aesthetically and functionally useful for the Serbian antag in filling this category instead of just converting one of the other. You are treating this as a shipside-only thing when it affects the balance of a major antags arsenal and access to a rifle that is very effective in fighting heavily armored opponents. The serbians are already in a weird position having to use a excel Makarov as their primary .35, please don't make it so they will now have to start carrying frozenstar bulls or excel vintorez. it ruins an antag balance just for some strange weapon change. it'd be like changing the frozenstar cheap AK to a .40 because we have like 5 AK skins.

**Comment by maikilangiolo** [comment]
> this is a specific antag faction weapon l
> fulfilling a heavy hitting armor piercing role

The Kovacs is commonly available in the vendor, but the Serbs also have the amr and the RPG (one essentially ignoring armor and the other bypassing heavy ballistic and melee armor). 

The Kovacs also deals more damage per shot than rifles, which also means better anti-armor performance than the AK (therefore still being useful). However, it is less efficient than the (now buffed) vintorez and Kadmin. 
It's advantage is the scope and high damage, offset by the recoil, while the vintorez and Kadmin have high damage past armor and excellent recoil.

**Comment by Firefox13** [comment]
It makes more sense for Vintorez to be .20 than Kovacs as it's based off, take a guess, a low-caliber (9×39mm) armour-piercing rifle

**Comment by Rexdyer1** [comment]
> The Kovacs is commonly available in the vendor, but the Serbs also have the amr and the RPG (one essentially ignoring armor and the other bypassing heavy ballistic and melee armor).
...alright I'll address this first. those are heavy specialist weapons that are not very effective in quick drawn out firefights with someone approaching. they do tons of damage but have a terrible reload. It does not help with giving a good weapons platform for the serbs to do good AP and decent damage against armor such as the Ironhammer wintermute or frozenstar bull. It fills a specific midrange niche that does good AP without being a heavy specialist weapon. It also creates two .30 rifles for the serbian arsenal and the vendor. if the Kovacs ability balance-wise is a big issue then it'd be better to just remove it from the vendor in the first place and just have the Serb AK instead. This creates 2 serbian .30 rifles in the arsenal and removes the AP rifle role from the serbian armory...also we have like 5 AK rifle varients. I think its acceptable to have the kovacs keep its similar role due to its importance as a serb specific gun filling the mercenaries need for an AP scoped rifle.

**Comment by maikilangiolo** [comment]
> It makes more sense for Vintorez to be .20 than Kovacs as it's based off, take a guess, a low-caliber (9×39mm) armour-piercing rifle

Also a reason of the change. The vintorez uses a small armor-piercing caliber, and the Kovacs is the FAL, which uses the full-power 7.62mm NATO.

**Comment by Rexdyer1** [comment]
is the gun ban still in place? because if you are all still insistent on this change I'm gonna need to make a reskinned alt bull for the serbs to make up for this lost .20 rifle niche.

**Comment by maikilangiolo** [comment]
> is the gun ban still in place? because if you are all still insistent on this change I'm gonna need to make a reskinned alt bull for the serbs to make up for this lost .20 rifle niche.

There's no need for it. I can only assume you did not listen but rather wanted to insist on your point of view. The Kovacs would work perfectly fine to defeat heavier armor with this change. It is simply less efficient than vintorez (which loses less damage) and has higher recoil (while dealing more damage).

**Comment by zombiemimic** [comment]
This is depriving the .20 caliber of its users, creating yet another .30 rifle. In addition, atomize your PRs.

**Comment by ValoTheValo** [comment]
> is the gun ban still in place? because if you are all still insistent on this change I'm gonna need to make a reskinned alt bull for the serbs to make up for this lost .20 rifle niche.

i have another serbian .20 assault carbine sprite that was rejected due to being made at the same time as the kovacs and serving a simmilar niche

**Comment by maikilangiolo** [comment]
> made at the same time as the kovacs and serving a simmilar niche

If you can convince Firefox, you can tell me you have my approval for this and I will handle the ugly, boring part of giving it stats. 

However, keep in mind two things. One, it's not strictly necessary the Serbians need this .20 assault carb (though the only small, easily carried gun they have is the zoric). Two, .20 is a well developed and varied caliber: revolver, assault rifle, *unique* carbine + UGL, sniper, bolt gun (technically also sniper). 

It's gonna be tricky not to create a wintermute, but like small or bulldog, but like without UGL

**Comment by ErisMeleeEnjoyer** [comment]
aw man this sure is a heated discussion, i am so glad that thanks to it no one will notice all the shadow nerfs I hid in my space weather PR

**Comment by Rexdyer1** [comment]
if it doesn't involve guns most people won't click on the PR itself unless bored sadly

**Pull request merged**